### ISSUE TITLE
Got the LV2 URI wrong

### ISSUE THREAD

**Issue by jpcima** [jpcima/string-machine#26]
Following the compatibility break in the dev version, it needs to update plugin URI.
I want to append `-mk2` as a prefix. (URL `http://jpcima.sdf1.org/lv2/string-machine-mk2`)

**Problem:** I thought to have done this in source, but mistook the variable, and instead I changed **homepage**.

Location of problem:
https://github.com/jpcima/string-machine/blob/1240f54c507c5bff3fa15ffd095d045c8be5b6f6/plugins/string-machine/meta/DistrhoPluginInfo.h#L6

**Comment by jpcima** [comment]
:man_facepalming:
(issue was lacking an adequate picture, now fixed)

**Comment by trebmuh** [comment]
:smile:

**Comment by jpcima** [comment]
I'm going to break the parameter compatibility...
There isn't a choice anyway, given that LV2 mandates this by standard.

In VST, I guess I have to change UniqueID.
The compatibility break is maybe why @sub26nico has met trouble in Reaper in dev versions.
It was changed some times during dev process.

AFAIK, there isn't a way to make a generic converter to migrate LV2 presets.
All hosts are using their own formats. Correct me if I'm wrong...

**Comment by Baggypants** [comment]
> AFAIK, there isn't a way to make a generic converter to migrate LV2 presets. All hosts are using their own formats. Correct me if I'm wrong...

This is correct. There is this thing https://gitlab.com/Jofemodo/preset2lv2 which might help.

**Comment by jpcima** [comment]
@Baggypants thanks, I'm sure yet how, but I'll check

The change of URI is enacted now for some time; an update tool for presets is feasible, I think to make it on select hosts in case there is some user demand.

In an ideal world, there would exist a tool which can handle all formats of Jalv, Zynthian, Carla, Ardour, etc..
Really a very lacking ability in the LV2 world IMO, which VST has permitted for a long time.
